Content.Server/GridPreloader/GridPreloaderSystem.cs

```diff
@@ -24,6 +24,12 @@ public sealed class GridPreloaderSystem : SharedGridPreloaderSystem
     [Dependency] private readonly IPrototypeManager _prototype = default!;
     [Dependency] private readonly SharedTransformSystem _transform = default!;
 
+    // TODO: use the efficient thing in initialize
```

Comment:
just do it its like 3 lines

Reply:
idk how to async map loading

Reply:
i was talking about the cvar being a getter instead of updated with onvaluechanged idk WTF ur talking about pro

Reply:
oh i thought i meant efficient thing as in rewrite the entire system :trollface: